pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[project]
+name = "ansible-bender"
+authors = [
+  {name = "Tomas Tomecek", email = "tomas@tomecek.net"},
+]
+description = "A tool which builds container images using Ansible playbooks"
+readme = "README.md"
+requires-python = ">= 3.6"
+keywords = [
+  "ansible",
+  "containers",
+  "linux",
+  "buildah",
+]
+license = {text = "MIT"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Software Development",
+    "Topic :: Utilities",
+]
+dependencies = [
+    "PyYAML",
+    "tabulate",
+    "jsonschema",
+    "setuptools",
+]
+dynamic = ["version"]
+
+[project.urls]
+homepage = "https://github.com/ansible-community/ansible-bender"
+documentation = "https://ansible-community.github.io/ansible-bender/build/html/index.html"
+
+[project.optional-dependencies]
+testing = [
+  "pytest",
+  "flexmock",
+  "pytest-cov",
+]
+
+[project.scripts]
+ansible-bender = "ansible_bender.cli:main"
+
+# reference for usage:
+# https://github.com/pypa/setuptools_scm/#pyprojecttoml-usage
+[tool.setuptools_scm]
+
+# references for minimum version selection:
+# setuptools: https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+# setuptools-scm: https://github.com/pypa/setuptools_scm/#pyprojecttoml-usage
+[build-system]
+requires = ["setuptools>=61", "setuptools-scm[toml]>=6.2", "wheel"]
```

Comment:
I just tested this PR locally in a fedora:36 container and it worked fine with `python3-setuptools-59.6.0-3.fc36.noarch` so I'm thinking that maybe we don't need to pin the version of setuptools

Reply:
fascinating, even works with a very old setuptools - `python3-setuptools-53.0.0-12.el9.noarch`

I suspect it's because `build` creates a venv and installs new setuptools inside: sadly the output doesn't say which versions

Reply:
Yes, build creates its own env but you can pass a CLI option to prevent it from managing the env.

Reply:
Note that ancient setuptools wouldn't know how to read `pyproject.toml`.